xonsh/tools.py

```diff
@@ -428,16 +428,59 @@ def print_exception(msg=None):
     if 'XONSH_SHOW_TRACEBACK' not in env:
         sys.stderr.write('xonsh: For full traceback set: '
                          '$XONSH_SHOW_TRACEBACK = True\n')
```

Comment:
@adqm I agree, I was a bit hasty and didn't see this note until now.

By the way, it seems that this check is rarely (if ever) triggered, because `XONSH_SHOW_TRACEBACK` is set to a default value of `False` (see `DEFAULT_VALUES` in `xonsh/environ.py`). Therefore, user with an empty `~/.xonshrc` will not see this tip.